src/Effects/Index.ts

```diff
@@ -0,0 +1,26 @@
+// Token-related effects
+export {
+  getTokenDetails,
+  getTokenPrice,
+  getTokenPriceData,
+  fetchTokenDetails,
+  fetchTokenPrice,
```

Comment:
I see we use the effect function (i.e. ones starting with `get`) in the indexer logic and the `fetch` functions only seem to be used for testing.

Is the separate testing of `fetch` functions necessary here?